gno.land/pkg/sdk/vm/gas_test.go

```diff
@@ -70,8 +70,8 @@ func TestAddPkgDeliverTx(t *testing.T) {
 
 	assert.True(t, res.IsOK())
 
-	// NOTE: let's try to keep this below 150_000 :)
-	assert.Equal(t, int64(141_197), gasDeliver)
+	// NOTE: let's try to keep this bellow 150_000 :)
+	assert.Equal(t, int64(225959), gasDeliver)
```

Comment:
The comment is now false.
I’m happy to increase our expected goal, but it seems this PR shouldn’t have such  a significant impact.
Can anyone find some performance quick wins.

Reply:
After the package is added via vm/keeper.AddPackage and m.RunMemPackage(), we need to process the storage deposit. This involves reading the realm object from the store and writing both the updated realm object and the sum of the storage changes back to the store. This is the source of the additional gas cost.

I've implemented some optimizations to eliminate redundant reads, which reduced the gas usage to 193,471. However, further reducing the gas cost of writing to the store would require more complex changes to the call flow and the tightly coupled logic between storage deposit calculation and m.RunMemPackage().

Given that this is a one-time action during package addition, such aggressive gas optimization isn't strongly recommended.